app/src/main/java/com/bnyro/contacts/ui/components/editor/TextFieldGroup.kt

```diff
@@ -0,0 +1,81 @@
+package com.bnyro.contacts.ui.components.editor
+
+import androidx.annotation.StringRes
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.rounded.Add
+import androidx.compose.material3.FilledTonalButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.snapshots.SnapshotStateList
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.dp
+import com.bnyro.contacts.obj.TranslatedType
+import com.bnyro.contacts.obj.ValueWithType
+
+@Composable
+fun SnapshotStateList<MutableState<ValueWithType>>.TextFieldGroup(
```

Comment:
```suggestion
fun TextFieldGroup(
```

Reply:
I would prefer passing this as a normal argument, I don't think it's common to use extension functions for composables like that.